padle/padle.py

```diff
@@ -0,0 +1,801 @@
+                cur_streak += 1
+                wins += 1
+            else:
+                if cur_streak > max_streak:
+                    max_streak = cur_streak
+                if day != await self.config.num_days():
+                    cur_streak = 0
+
+        if cur_streak > max_streak:
+            max_streak = cur_streak
+        all_monsters_guessed = []
+        for key, value in all_guesses.items():
+            all_monsters_guessed.extend(value)
+        mode = max(set(all_monsters_guessed), key=all_monsters_guessed.count)
+        dbcog = await self.get_dbcog()
+        m = dbcog.get_monster(mode)
+        m_embed = EmbedView(
+            EmbedMain(
+                title=f"{ctx.author.name}'s PADle Stats",
+                description=(f"**Games Played**: {played}\n"
+                             "**Win Rate**: {:.2%}\n".format(wins / played) +
+                             f"**Current Streak**: {cur_streak}\n"
+                             f"**Max Streak**: {max_streak}\n"
+                             f"**Favorite Guessed Monster**: {MonsterHeader.menu_title(m).to_markdown()}")),
+            embed_thumbnail=EmbedThumbnail(
+                MonsterImage.icon(m.monster_id))).to_embed()
+        await ctx.send(embed=m_embed)
+
+    async def get_past_padle_embed(self, ctx, day: int):
+        dbcog = await self.get_dbcog()
+        daily_scores = await self.config.save_daily_scores()
+        info = daily_scores[day - 1]
+        stats = info[1]
+        giveups = 0
+        completes = 0
+        average = 0
+        for item in stats:
+            if item == "X":
+                giveups += 1
+            else:
+                completes += 1
+                average += int(item)
+        guess_monster = dbcog.get_monster(int(info[0]))
+        embed = EmbedView(
+            EmbedMain(
+                title="PADle #{} Stats".format(day)),
+            embed_thumbnail=EmbedThumbnail(
+                MonsterImage.icon(guess_monster.monster_id))).to_embed()
+        if completes == 0:
+            embed.description = ("**" + MonsterHeader.menu_title(guess_monster).to_markdown() + "**\n" +
+                                 "**Total Wins**: {}\n**Total Losses**: {}\n**Win Rate**: 0%\n"
+                                 "**Average Guess Count**: 0").format(completes, giveups)
+        else:
+            embed.description = ("**" + MonsterHeader.menu_title(guess_monster).to_markdown() + "**\n" +
+                                 "**Total Wins**: {}\n**Total Losses**: {}\n**Win Rate**: {:.2%}\n"
+                                 "**Average Guess Count**: {:.2f}").format(completes, giveups,
+                                                                           completes / (completes + giveups),
+                                                                           (average / completes))
+        return embed
+
+    # this takes a long time to run
```

Comment:
Remember to remove commented out code before your final commit unless you wanna keep it for future stuff!

Reply:
should I keep the code in, commented, if we wanted to remake the full list of monsters again in the future?